americangut/results_utils.py

```diff
@@ -7,6 +7,8 @@
 from collections import defaultdict
 from functools import partial
 
+import matplotlib.pyplot as plt
```

Comment:
This works fine on a local system but can fail on remote if forwarding is not available. Does it make sense to call ```plt.switch_backend('egg')``` after this line (or chose the backend before importing pyplot) to avoid that error?